modules/final_radiological_review/templates/menu_final_radiological_review.tpl

```diff
@@ -69,8 +69,8 @@
                         <div class="col-sm-12 col-md-4">{$form.Final_Review_Results.html}</div>
                     </div>
                     <div class="form-group col-sm-4">
-                        <label class="col-sm-12 col-md-8">{$form.Exclusionary_Status.label}</label>
-                        <div class="col-sm-12 col-md-4">{$form.Exclusionary_Status.html}</div>
+                        <label class="col-sm-12 col-md-8">{$form.Project.label}</label>
+                        <div class="col-sm-12 col-md-4">{$form.Project.html}</div>
```

Comment:
Will this section cause any issues when $form.Project.label and $form.Project.html are not defined?